collection/Dungeon Dudes; Sebastian Crowe's Guide to Drakkenheim.json

```diff
@@ -2865,248 +3440,3632 @@
+			"source": "SCGtD",
+			"page": 153,
+			"className": "Bard",
+			"classSource": "PHB",
+			"subclassShortName": "Doomsayers",
+			"subclassSource": "SCGtD",
+			"level": 3,
+			"entries": [
+				"Bards of this college revel in the fear and despair of others. They have foreseen the end of times, and use their gifts of prophecy to instill terror in those who hear their words. These bards command an audience like any other, but their craft is that of a mad prophet, and their poems and songs echo dissonant rattlings of hopelessness and suffering. Those who hear the mad ramblings are stricken with fear, lose their wills, and fall into madness.",
+				{
+					"type": "refSubclassFeature",
+					"subclassFeature": "The End is Nigh|Bard|PHB|Doomsayers|SCGtD|3|SCGtD"
+				},
+				{
+					"type": "refSubclassFeature",
+					"subclassFeature": "Dreadful Words|Bard|PHB|Doomsayers|SCGtD|3|SCGtD"
+				}
+			]
+		},
+		{
+			"name": "The End is Nigh",
+			"source": "SCGtD",
+			"page": 153,
+			"className": "Bard",
+			"classSource": "PHB",
+			"subclassShortName": "Doomsayers",
+			"subclassSource": "SCGtD",
+			"level": 3,
+			"entries": [
+				"Your words strike fear into the hearts of those who bear witness to your doomsaying. Whenever you make a Charisma (Intimidation) check, you may treat any roll of 9 or lower as if you had rolled a 10."
+			]
+		},
+		{
+			"name": "Dreadful Words",
+			"source": "SCGtD",
+			"page": 153,
+			"className": "Bard",
+			"classSource": "PHB",
+			"subclassShortName": "Doomsayers",
+			"subclassSource": "SCGtD",
+			"level": 3,
+			"entries": [
+				"As a bonus action, you can expend one use of your Bardic Inspiration and choose one creature you can see within 60 feet of you. The target takes psychic damage equal to the roll of your Bardic inspiration die and must make a Wisdom saving throw against your spell save DC. On a failed saving throw, the target becomes frightened of you until the start of your next turn."
+			],
+			"foundrySystem": {
+				"activation": {
+					"type": "bonus",
+					"cost": 1
+				},
+				"save": {
+					"ability": "wis",
+					"scaling": "cha"
+				}
+			}
+		},
+		{
+			"name": "Prophetic Foresight",
+			"source": "SCGtD",
+			"page": 153,
+			"className": "Bard",
+			"classSource": "PHB",
+			"subclassShortName": "Doomsayers",
+			"subclassSource": "SCGtD",
+			"level": 6,
+			"entries": [
+				"As a bonus action, you can expend one use of your Bardic Inspiration and choose one creature you can see within 60 feet of you. The target takes psychic damage equal to the roll of your Bardic inspiration die and must make a Wisdom saving throw against your spell save DC. On a failed saving throw, the target becomes frightened of you until the start of your next turn."
+			],
+			"foundrySystem": {
+				"activation": {
+					"type": "bonus",
+					"cost": 1
+				},
+				"save": {
+					"ability": "cha",
+					"scaling": "cha"
+				},
+				"uses": {
+					"max": "floor(@prof / 2)",
```

Comment:
For some reason, when imported, this still gives 2.5 uses instead of 2 when PB is 5. Any idea why that would be?